src/views/Excel/private/constants.ts

```diff
@@ -0,0 +1,29 @@
+import type { HeaderItemProps } from '@/types'
+
+export const header: HeaderItemProps[] = [
+  {
+    key: 'id',
+    width: 20,
+    ignore: false
```

Comment:
If we set the `ignore` is an optional field, we can remove it from the object, cause making it `undefined` works the same.